config/serverless.yml

```diff
@@ -1 +1,2 @@
 xpack.fleet.enableExperimental: ['fleetServerStandalone']
+xpack.fleet.disableILMPolicies: true
```

Comment:
@elastic/fleet I have some questions on the naming, I am wondering if we should introduce something like `xpack.fleet.internalDisableILMPolicies` as end user should probably not use that settings? (open to better naming too)

Reply:
we do have the `dev` section of the config but I guess this is different. If we may have more config items like this we could have an `internal` section e.g `xpack.fleet.internal.disableILMPolicies`   

Finally I think `ILMPoliciesDisabled` may be better however the capitalization looks so strange 😆 Naming things 🤦

Reply:
I like the original suggestion of `xpack.fleet.internal.disableILMPolicies` because it avoids casing weirdness. Seems like this aligns better with other configuration values.

Reply:
> I like the original suggestion of xpack.fleet.internal.disableILMPolicies because it avoids casing weirdness. Seems like this aligns better with other configuration values.

Yes I will make the change the `ILM...` casing seems really weird